src/WebUI/Components/DeploymentsComponent.tsx

```diff
@@ -85,7 +104,10 @@ export class DeploymentsComponent extends BaseComponent<IDeploymentsComponentPro
             pods: DeploymentsComponent._getPodsText(replica.status.availableReplicas, replica.status.replicas),
             statusProps: DeploymentsComponent._getPodsStatusProps(replica.status.availableReplicas, replica.status.replicas),
             showRowBorder: (replicaSetLength === (index + 1)),
-            deployment: deployment
+            deployment: deployment,
+            //todo :: how to display images of all containers in a replica pod

```

Comment:
change the todo from how to display images of all containers in a replica pod
to
should we show all containers in a replica pod